### ISSUE TITLE
Anyone integrated it with Netlify CMS?

### ISSUE THREAD

**Issue by username1290** [tomjohnson1492/documentation-theme-jekyll#159]

**Comment by briandominick** [comment]
Yeah, just point a new Netlify project toward your Git repo and give it a build command (not serve), like `bundle exec jekyll build`. Then sit back and magic.

**Comment by username1290** [comment]
so, downloaded zip, unzipped it
removed gemfiles as stated in the docs
added files to a repo
added the command ^^^^ in Netlify's build command input field
and magic 😄 Deployed on Netlify 👍 
Thanks!
